otherlibs/systhreads/Makefile

```diff
@@ -18,6 +18,10 @@ ROOTDIR=../..
 include $(ROOTDIR)/Makefile.config
 include $(ROOTDIR)/Makefile.common
 
+OC_CPPFLAGS += -I$(ROOTDIR)/runtime
```

Comment:
You should also update the `.depend` target in this file, which can be simplified thanks to this definition.